src/api/app/models/owner.rb

```diff
@@ -87,7 +87,7 @@ def self.search(params, obj)
   end
 
   def self.find_assignees(rootproject, binary_name, limit = 1, devel = true, filter = %w(maintainer bugowner), webui_mode = false)
-    projects = rootproject.expand_all_projects
+    projects = rootproject.expand_all_projects(allow_remote_projects: false)
```

Comment:
It will change behavior: If one of the linked projects is remote it won't crash ;-)